test/aggregate.js

```diff
@@ -20,7 +20,7 @@ cursor = db['aa0d6eda-147d-469a-a4ef-075872701060-event'].aggregate([{
     restrictSearchWithMatch: {
```

Comment:
This whole file should be removed, it's obsolete.

Reply:
addressed here: https://github.com/digitalbazaar/bedrock-ledger-consensus-continuity/pull/259/commits/18b0f6ad18ea4f9912621f59e27dda2d271a3c7c